src/main/java/io/github/railroad/config/SyntaxConfig.java

```diff
@@ -1,25 +1,20 @@
 package io.github.railroad.config;
 
-import java.io.File;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-
-import org.json.JSONArray;
-import org.json.JSONObject;
-import org.json.JSONTokener;
-
 import io.github.railroad.Railroad;
 import io.github.railroad.debugger.syntax.EnumSyntaxType;
 import io.github.railroad.debugger.syntax.SyntaxObject;
 import io.github.railroad.utility.FileUtils;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.json.JSONTokener;
+
+import java.io.File;
+import java.util.*;
 
 public class SyntaxConfig extends AbstractConfig {
 
-	public List<SyntaxObject> languages = new ArrayList<SyntaxObject>();
+	public List<SyntaxObject> languages = new ArrayList<>();
```

Comment:
same here. why are you leaving it like that. just remove the <> completely?

Reply:
<> is required, but inserting the value is redundant. You can take IJ's word for it, too. I do this too, IJ just picks up on it.

Reply:
its most certainly not required. you either leave it raw or keep the generics all there, leaving it blank makes no sense at all

Reply:
I mean, yes, but there's also no reason to *keep* it. Don't take it from me; IJ's suggestions are submitted by experts, look it up.